test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -265,8 +268,10 @@ public void run(String[] args) {
             Point targetPoint = new Point(x + w / 2, y + h / 2);
 
             Robot robot = new Robot();
+            robot.setAutoWaitForIdle(true);
             robot.mouseMove(sourcePoint.x, sourcePoint.y);
-            robot.mousePress(InputEvent.BUTTON1_MASK);
+            Thread.sleep(50);
```

Comment:
```suggestion
            robot.delay(50);
```
Using `Robot.delay` is more common in tests, and it doesn't throw `InterruptedException`.

Reply:
Thank you for your suggestions. The changes have been incorporated.